task1/task1.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+file="$1"
+folder="$2"
```

Comment:
There is no need to add inverted commas. Just the dollar sign will do the work, in mostly all scenarios.